tests/shared/src/test/scala/scala/meta/tests/parsers/DottyEnumSuite.scala

```diff
@@ -0,0 +1,583 @@
+package scala.meta.tests.parsers
+
+import org.scalatest.FunSuite
+import org.scalatest._
+import scala.meta._
+import scala.meta.dialects.Dotty
+
+class DottyEnumSuite extends FunSuite {
```

Comment:
Let's reduce the verbosity of these tests. If a test can fit on a single line, write it on a single line. E.g.
```scala
// prefer
checkError("sealed enum Foo {}")

// to
checkError(
  "sealed enum Foo {}"
)
```
```scala
// prefer
checkError("enum Foo { lazy case A }")

// to
checkError(
  """
    |enum Foo {
    | lazy case A
    |}
  """.stripMargin
)
```

Group single line related test together. E.g.
```scala
checkError("enum Foo { final case A }")
checkError("enum Foo { lazy case A }")
```
This comment applies to this file and `DottyEnumSyntacticSuite.scala`

Reply:
Ok I will change that. Do I put the source code on one line for the tests with expected tree too ?

Reply:
I see. I made changes, tell me if it is fine now.
I put on one line the code if it was very small (e.g. "enum Foo { case A }") even if it was in a test with expected tree because one or two were already in this form.